releasenotes/notes/fix-codefresh-integration-563b8075e71881e4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    CI visibility: added BITBUCKET_GIT_HTTP_ORIGIN environment variable and added Codefresh integration
```

Comment:
```suggestion
  - |
    CI visibility: Adds BITBUCKET_GIT_HTTP_ORIGIN environment variable
  - |
    CI visibility: Adds CodeFresh integration
```

Reply:
Good idea! I will split up the features in the release notes as suggested.